adam-core/src/test/scala/org/bdgenomics/adam/rdd/variant/VariantContextRDDSuite.scala

```diff
@@ -210,7 +210,7 @@ class VariantContextRDDSuite extends ADAMFunSuite {
     val rdd: VariantContextRDD = sc.loadVcf(freebayesVcf)
 
     implicit val tFormatter = VCFInFormatter
```

Comment:
How do we provide `conf` to the `VCFInFormatter`?

Reply:
It gets it through the `rdd.context.hadoopConfiguration`.